src/string.rs

```diff
@@ -0,0 +1,171 @@
+use rand::seq::SliceRandom;
```

Comment:
You shouldn't use rand package because the whole package using [simplerand](https://github.com/PumpkinSeed/simplerand). It is because the rand package is an overkill for this library. Also if you are using rand package and I add the custom seed value functionality to the code-base (one of the requested feature), it won't work for string functions, because it will use the rand package's seed value. 

Please use simplerand, or write a wrapper/helper function for that in the misc.rs.